synapse/cortex.py

```diff
@@ -1014,8 +1014,15 @@ async def initServiceStorage(self):
         await self._bumpCellVers('cortex:storage', (
             (1, self._storUpdateMacros),
             (2, self._storLayrFeedDefaults),
+            (3, self._viewNomergeToProtected),
         ), nexs=False)
 
+    async def _viewNomergeToProtected(self):
+        for view in self.views.values():
+            nomerge = view.info.get('nomerge', None)
+            await view.setViewInfo('protected', nomerge)
+            await view.setViewInfo('nomerge', None)
```

Comment:
Could we pop this from the view info?

Reply:
Passing `None` as the value to `setViewInfo()` pops it from the hive dict. Doing `await view.info.pop('nomerge')` didn't work (I think because the underlying hivedict is not nexus enabled).